pywinauto/application.py

```diff
@@ -1044,10 +1044,10 @@ def wait_cpu_usage_lower(self, threshold=2.5, timeout=None, usage_interval=None)
         if timeout is None:
             timeout = Timings.cpu_usage_wait_timeout
 
-        start_time = time.time()
+        start_time = time.clock()

```

Comment:
Alternatively we could use `timeit` module. It is supposed to be more reliable in terms of calculating elapsed time on different operating systems. See https://docs.python.org/2/library/timeit.html#timeit.default_timer